Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -46,9 +46,34 @@ class SettingsViewController: UIViewController, ToolbarViewProtocol, UITableView
         signOutButton.addTarget(self, action: "didClickLogout", forControlEvents: UIControlEvents.TouchUpInside)
     }
 
-    // Referenced as button selector.
+    // A temporary delegate which merely updates the displayed email address on
+    // succesful Firefox Accounts sign in.
+    private class Delegate: FxASignInViewControllerDelegate {
```

Comment:
Why do these need to be in their own class? Can't they be part of the `SettingsViewController` directly?

I think it makes more sense to name these:

```
func signInViewControllerDidCancel(vc: FxASignInViewController)
func signInViewController(vc: FxASignInViewController, didSignIn account: JSON)
```
And include them directly in SettingsViewController. It is very uncommon to do that extra level of indirection.